pushy/src/main/java/com/eatthepath/pushy/apns/ApnsChannelPool.java

```diff
@@ -220,6 +222,7 @@ private void discardChannel(final Channel channel) {
         this.idleChannels.remove(channel);
         this.allChannels.remove(channel);
 
+        log.debug("Discarded channel {}", channel);
```

Comment:
Same question here.  Is this serious enough, or does it happen infrequently enough for info/warn/error?

Reply:
Again, I think this is expected and doesn't rise to the level of an "important lifecycle event." Let's leave it at `debug`.